corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -67,6 +67,20 @@ FormplayerFrontend.reqres.setHandler('clearMenu', function () {
     $('#menu-region').html("");
 });
 
+$(document).bind("ajaxStart", function(){
+    var requestElements = $(".formplayer-request");
+    for (var i = 0; i < requestElements.length; i++ ) {
+        requestElements.css('pointer-events', 'none');
```

Comment:
by having a class that you can toggle on and off, it'll be easier to specify what want a disabled formplayer button to look like. `$(".formplayer-request").addClass('formplayer-btn-disabled')` and then in the other one  `$(".formplayer-request").removeClass('formplayer-btn-disabled')`. Then just set the css props on the formplayer-btn-disabled class in a css/less file